flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionITCase.java

```diff
@@ -1038,9 +1112,9 @@ private static List<TestSetSpec> toStringCasts() {
                                 DEFAULT_NEGATIVE_DOUBLE,
                                 String.valueOf(DEFAULT_NEGATIVE_DOUBLE))
                         .fromCase(DATE(), DEFAULT_DATE, "2021-09-24")
-                        // https://issues.apache.org/jira/browse/FLINK-17224 Currently, fractional
-                        // seconds are lost
-                        .fromCase(TIME(5), DEFAULT_TIME, "12:34:56")
+                        .fromCase(TIME(5), DEFAULT_TIME, "12:34:56.123")
```

Comment:
This shows, sth is wrong with `TIME(5)`.

Reply:
May be we still should put comment like micro and nano are not supported for `TIME` yet?